jobs/FunctionTest/FunctionTest.groovy

```diff
@@ -54,12 +54,23 @@ def functionTest(String test_name, String TEST_GROUP, Boolean RUN_FIT_TEST, Bool
             try{
                 timeout(90){
                     // run test script
-                    sh '''#!/bin/bash -x
-                    pushd build-config
-                    ./build-config
-                    popd
-                    ./build-config/test.sh
-                    '''
+                    def repos = env.REPOS_UNDER_TEST.tokenize(',')
```

Comment:
just a slight note:
PR Gate for this PR will never test those functionalities, becuase this is a on-build-config PR, and REPOS_UNDER_TEST will not contains image-service, so this PR's PR Gate test will still use default test.sh.in